lib/gc/ExecutionEngine/OpenCLRuntime/OpenCLRuntimeWrappers.cpp

```diff
@@ -390,6 +390,14 @@ extern "C" OCL_RUNTIME_EXPORT void gpuMemFree(GPUCLQUEUE *queue, void *ptr) {
   }
 }
 
+extern "C" OCL_RUNTIME_EXPORT void gpuMemCopy(GPUCLQUEUE *queue, void *dst,
+                                              void *src, uint64_t size) {
+  auto func = queue->ext_table_ ? queue->ext_table_->enqueneMemcpy
```

Comment:
Sorry, it is a typo of mine: `enquene` should be `enqueue`. Shall we fix it by the way?

Reply:
Didn't notice this, thanks. I'll add a fix in this PR.